rabbitmq/json-queues.yml

```diff
@@ -0,0 +1,147 @@
+type: json
+data:
+  url: /api/queues
+  server: ${SPM_MONITOR_RABBITMQ_API_URL}
+  basicHttpAuthUsername: ${SPM_MONITOR_RABBITMQ_BASICAUTH_USERNAME}
+  basicHttpAuthPassword: ${SPM_MONITOR_RABBITMQ_BASICAUTH_PASSWORD}
+
+observation:
+  - name: queues
+    metricNamespace: rabbitmq
+    path: $.[?(@.name=${name} && @.vhost=${vhost})]
+
+    metric:
+      - name: queues.consumers.active
+        source: active_consumers
+        type: long_gauge
+        label: active consumers
+        description: Number of active consumers, consumers that can immediately receive any messages sent to the queue
+
+      - name: queues.consumers
+        source: consumers
+        type: long_gauge
+        label: consumers
```

Comment:
I also see another metric in `json-overview.yml` with same label as `consumers`.  Do they refer to the same value?

Reply:
Not the same, should we use prefix? is there any collision if we use same name?

Reply:
There won't be collision with labels. But user will see same label for both metrics. Yes we can use some prefix to differentiate.

Reply:
The are the same concept, but for value it may be different. in json-overview it will count total consumers while in queues, i count only consumer related to that queue